platform/darwin/src/MGLVectorTileSource.mm

```diff
@@ -89,17 +89,17 @@ @implementation MGLVectorTileSource (Private)
  https://www.mapbox.com/vector-tiles/mapbox-streets-v8/
  */
 static NSArray * const MGLMapboxStreetsLanguages = @[
-    @"ar", @"de", @"en", @"es", @"fr", @"ja", @"ko", @"pt", @"ru", @"zh",
-    @"zh-Hans",
+    @"ar", @"de", @"en", @"es", @"fr", @"ja", @"ko", @"pt", @"ru", @"vi",
```

Comment:
`-[NSExpression mgl_expressionLocalizedIntoLocale:]` has logic to fall back from `zh-Hant` to `zh` if the style happens to be using Streets source v7.